README.md

```diff
@@ -152,48 +152,25 @@ kubectl apply -f https://raw.githubusercontent.com/fluent/fluentbit-operator/mas
 
 #### Deploy Fluent Operator with Helm
 
-> Note: For the helm based install, Helm v3.2.1 or higher is needed.
-
-The Fluent Bit section of the Fluent Operator supports different CRI `docker`, `containerd`,  and `CRI-O`. 
-`containerd` and `CRI-O` use the `CRI Log` format which is different with `docker`, they requires additional parser to parse JSON application logs. You should set different `containerRuntime` depending on your container runtime.
-
-The default runtime is docker, you can choose other runtimes as follows.
-
-If your container runtime is `containerd` or  `cri-o`, you can set the `containerRuntime` parameter to `containerd` or `crio`. e.g.
+> NOTE: For the helm based install, Helm v3.2.1 or higher is needed.
 
 ```shell
-helm install fluent-operator --create-namespace -n fluent charts/fluent-operator/  --set containerRuntime=containerd
-```
+export FLUENT_OPERATOR_CONTAINER_RUNTIME="containerd" # or "cri-o", "docker" depending on the container runtime being used (see `values.yaml`)
 
-Install through the online chart link:
-
-```shell
-helm install fluent-operator --create-namespace -n fluent https://github.com/fluent/fluent-operator/releases/download/< version >/fluent-operator.tgz
+helm repo add fluent https://fluent.github.io/helm-charts
```

Comment:
Use the `fluent/helm-charts` repo as a source for the Helm chart installation rather than a local copy of the chart.